### ISSUE TITLE
wrong link for understanding non text content wcag 2.2

### ISSUE THREAD

**Issue by StackOverflowIsBetterThanAnyAI** [w3c/wcag#3619]
Hey, on this page:
_https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html_

you have to navigate to: 
**'Short text alternative techniques for Situation B'**

You are referencing Technique H35, but you are using the **wrong** link.
It should be this one: 
**https://www.w3.org/WAI/WCAG22/Techniques/html/H35**

At the moment I get an error 404 message.

Kind regards, Michael

**Comment by mbgower** [comment]
@StackOverflowIsBetterThanAnyAI 
That link now appears to be resolved for me. If you are still finding an issue, please reopen and comment.

**Comment by giacomo-petri** [comment]
Hi @mbgower,

the technique has been removed with https://github.com/w3c/wcag/pull/2490, but the link is still available in the 1.1.1 understanding doc. 

I'm opening a PR to remove this link.
